test/regression/base/arithmetic/miniexample3.smt2

```diff
@@ -0,0 +1,8 @@
+(set-logic QF_LIA)
+(declare-fun x () Int)
+(declare-fun y () Int)
+(define-fun uninterp_mul ((a Int) (b Int)) Int (+ (* a b) 10))
+(assert (= x x))
```

Comment:
This seems useless. Please, keep the tests focused.

Reply:
Fixed